Cargo.lock

```diff
@@ -3350,9 +3350,9 @@ dependencies = [
 
 [[package]]
 name = "near-account-id"
-version = "1.0.0-alpha.1"
+version = "1.0.0-alpha.2"
```

Comment:
`1.0.0-alpha.2` now has some changes that previously have been in this PR, see https://github.com/near/near-account-id-rs/pull/14/